src/item.h

```diff
@@ -462,7 +462,8 @@ class item : public visitable<item>
         bool burn( fire_data &frd );
 
         // Returns the category of this item.
-        const item_category &get_category() const;
+        // If (insides == true) returns the category of item inside this item. I.e. "can of meat" would be food, instead of container.
+        const item_category &get_category( bool insides = false ) const;
```

Comment:
Please consider making this a separate function instead of adding a boolean parameter.

Reading `it.get_category( true )` doesn't tell the reader what the boolean means. It could related to anything, not just items in containers.

Reply:
Done.